### PR TITLE
Qa 2113 soft launch test automation update 1

### PR DESCRIPTION
**Pull request by bpierce01**
## Description
- Updated test data for DR39 
- Updated some facet selection new reflect new labels and behavior
- Updated text checking as necessary 

Test on qa-orange, using 'export APP_ENVIRONMENT=PROD_SOFT_LAUNCH' before launching the tests

Two existing failures:
- general_cohort_functions.spec: The text on the 2nd save message is wrong. There is a bug ticket on the PEAR board to fix. 
- header/navigation.spec: I haven't been able to fix this test. It is a problem on our end, the data portal is fine.

### CODE REVIEW

**Review by AmyCMP — COMMENTED** [review]
There are two test failures reported while running automation, more detailed report could be checked in here   
[Gauge Reports jan9th.pdf](https://github.com/NCI-GDC/gdc-frontend-framework/files/13879016/Gauge.Reports.jan9th.pdf)
[Gauge Report2 Jan 9th.pdf](https://github.com/NCI-GDC/gdc-frontend-framework/files/13879019/Gauge.Report2.Jan.9th.pdf)

**Comment by bpierce01** [comment]
> There are two test failures reported while running automation, more detailed report could be checked in here [Gauge Reports jan9th.pdf](https://github.com/NCI-GDC/gdc-frontend-framework/files/13879016/Gauge.Reports.jan9th.pdf) [Gauge Report2 Jan 9th.pdf](https://github.com/NCI-GDC/gdc-frontend-framework/files/13879019/Gauge.Report2.Jan.9th.pdf)

Yes the cohort bar failure is expected and there is a bug ticket for it. The header one I've been trying to fix for a while, on some environments it fails and on some it doesn't.

**Comment by AmyCMP** [comment]
> > There are two test failures reported while running automation, more detailed report could be checked in here [Gauge Reports jan9th.pdf](https://github.com/NCI-GDC/gdc-frontend-framework/files/13879016/Gauge.Reports.jan9th.pdf) [Gauge Report2 Jan 9th.pdf](https://github.com/NCI-GDC/gdc-frontend-framework/files/13879019/Gauge.Report2.Jan.9th.pdf)
> 
> Yes the cohort bar failure is expected and there is a bug ticket for it. The header one I've been trying to fix for a while, on some environments it fails and on some it doesn't.

@uc-pperumala do you have any concerns on the Header navigation failure?

**Review by AmyCMP — APPROVED** [review]
Except the two known issues, everything looks good to me...

**Review by uc-pperumala — APPROVED** [review]

File: holmes-py/specs/gdc_data_portal_v2/projects/table_download.spec [hunk 1]
```diff
@@ -43,6 +43,7 @@ tags: gdc-data-portal-v2, regression, projects, download
 * Verify that "TSV Projects Table from Projects" does not contain specified information
   |required_info|
   |-------------|
+  |GENIE        |
```
Comment (wteouchicago) [inline holmes-py/specs/gdc_data_portal_v2/projects/table_download.spec]: do you have GENIE twice?

Comment (bpierce01) [reply holmes-py/specs/gdc_data_portal_v2/projects/table_download.spec]: Haha yes it appears I do!

**Review by wteouchicago — APPROVED** [review]

**Review by craigrbarnes — APPROVED** [review]
Changes are fin

**Pull request merged**